R/utils.R

```diff
@@ -54,3 +54,21 @@ deparse_to_one_line <- function(x) {
          paste(deparse(x), collapse = ' '),
          perl = TRUE)
 }
+
+#' Catch the log header
+#' @return string
+#' @keywords internal
+#' @param level see \code{\link{log_levels}}
+#' @param namespace string
+catch_base_log <- function(level, namespace) {
+    orginal_appender <- log_appender()
```

Comment:
don't we need to pass `namespace` here and below as well to the `log_appender` calls?

Reply:
Connected with https://github.com/daroczig/logger/pull/96#discussion_r760845468. This is only a dummy setup to send everything to stderr.

Reply:
applied